src/lib.rs

```diff
@@ -2,9 +2,6 @@
 see https://github.com/aaronabramov/k9 for high level description of the library
 */
 
-#[allow(clippy::unknown_clippy_lints)]
-#[allow(clippy::unnested_or_patterns)]
```

Comment:
interesting! i think this one is flaky!
sometimes in nightly it would break, so i had to disable it it for nightly only and also disable the unknown warns (cause it wasn't on stable)

but yeah, if that works let's remove it!